### DISCUSSION TITLE
Welcome to Azure-Arc-Windows-Linux-Dashboard Discussions!

### DISCUSSION THREAD

**Discussion by weeyin83** [weeyin83/Azure-Arc-Windows-Linux-Dashboard#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sylve32** [comment]
is there a way we can create a resource graph in Azure that displays resources properties including the cores of the resource

**Reply by weeyin83** [reply]
There is a query within the dashboard that shows the number of cores on the Windows machines that are ESU enabled - [https://github.com/weeyin83/Azure-Arc-Windows-Linux-Dashboard/tree/main/queries#windows-esu-assignment-status](https://github.com/weeyin83/Azure-Arc-Windows-Linux-Dashboard/tree/main/queries#windows-esu-assignment-status).

Are you looking for something different?

**Reply by Sylve32** [reply]
okay but is there a way we can tabulate data or server properties information on a table in arc dashboard

**Reply by weeyin83** [reply]
Sorry I don't understand the ask, that information is available via a table in the dashboard, there is also a "download as csv" button if you want to take the information into an Excel workbook to use.

**Reply by Sylve32** [reply]
for example we have a server that is in Arc right, I would want to display the server properties in a tabular format

**Reply by weeyin83** [reply]
You can build the query you want and display it on the dashboard.  You have the ability to display query information on the dashboard in table, bar chart, or donut chart format. 

If you know what information you want to display I can help you build that query.

**Reply by Sylve32** [reply]
okay thanks, I wanted to create a query that will list whether there is  SSIS, SSRS, SSAS installed in a server

**Reply by weeyin83** [reply]
Unfortunately Azure Arc doesn't query the software that is installed on a server, you would need to enable the the [Inventory tracking feature first](https://learn.microsoft.com/azure/automation/automation-tutorial-installed-software), then you could build a query to list the software installed.

**Comment by Sylve32** [comment]
Okay thanks Sarah,

**Comment by Sylve32** [comment]
Hi Sarah,
I have a query that shows up the number of cores per each server and I am
presenting it in a pie chart view (donut view) and in the middle of the
donut its computing the percentage cores of each server. Can I have a query
that computes the number of cores per server not the percentage

**Comment by Sylve32** [comment]
Hi Sarah, 
Im working on enhancing my azure arc dashboard and i  would want it to display the memory and cpu and storage of a server deployed on Azure arc as below. I have been running my code to retain the information but wasnt successful. 
![image](https://github.com/weeyin83/Azure-Arc-Windows-Linux-Dashboard/assets/19346641/f64f8a60-e15c-4101-a302-ca6493d7d390)

**Reply by weeyin83** [reply]
Hey @Sylve32, not sure what you are having an issue with? Are you trying to pin new information to the dashboard but it's not saving?

**Reply by Sylve32** [reply]
Hi Sarah, 
So the above screenshot i attached, isn't mine, so I am trying to come up with something like that, but i am hitting a stumbling block on code.

Is there a set aside query to pull that information to a dashboard or do you have a query you can share to pull that information.

thanks

**Reply by weeyin83** [reply]
That's not actually a quick dashboard query in the screenshot, that's actually a link to an Azure Monitor workbook as there is quite a lot of data and data aggregation happening to display.   I would suggest you deploy the Azure Arc Jumpstart demo environment where you got that screenshot from and take a look at it in action to understand it it a bit better.  It's a great learning environment.

**Reply by Sylve32** [reply]
okay thanks Sarah
